bloom-housing/bloom
# fix: applying pass key to endpoints that were missing it

This PR addresses #4354 


## Description
Applying api pass key to the endpoints that were missing it

## How Can This Be Tested/Reviewed?
Double check you are still able to log in to the partner and public sites and just bop around to make sure happy paths still work

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
